server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/GitIntegrationController.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.server.web.controller.v2;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.oceanbase.odc.service.common.response.PaginatedResponse;
+import com.oceanbase.odc.service.common.response.Responses;
+import com.oceanbase.odc.service.git.GitIntegrationService;
+import com.oceanbase.odc.service.git.model.GitProvider;
+import com.oceanbase.odc.service.git.model.GitRepository;
+import com.oceanbase.odc.service.git.model.QueryGitRepositoryParams;
+import com.oceanbase.odc.service.git.model.VcsProvider;
+
+import io.swagger.annotations.ApiOperation;
+
+/**
+ * @author: liuyizhuo.lyz
+ * @date: 2024/8/9
+ */
+@RestController
+@RequestMapping("/api/v2/project/{projectId}/integration/repos")
```

Comment:
url style, {module}/resources/{id}/subresources/

-->

/api/v2/projects/{projectId}/gitRepos/


emm, what is the module name, collaboration？

Reply:
I changed it to `/api/v2/collaboration/projects/{projectId}/repos`.
How about this?

Reply:
not repos, but gitRepos

Reply:
okay, done.